src/test/kotlin/no/nav/tilleggsstonader/sak/IntegrationTest.kt

```diff
@@ -92,6 +93,8 @@ abstract class IntegrationTest {
     @Autowired
     protected lateinit var testoppsettService: TestoppsettService
 
+    val logger = LoggerFactory.getLogger(javaClass)
```

Comment:
Ikke at det gjør noe, men er det meningen at denne skulle med?

Reply:
Yep 😬 fikk med meg logger fra en PdlUtil eller noe slikt første gangen jeg brukte logger i testen. Tenkte det var fint å alltid ha en logger tilgjengelig i testene :) 
Mulig man egentlige burde fjerne den "globale loggern" for å unngå dette problemet.